addons/payment_test/tests/common.py

```diff
@@ -0,0 +1,78 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.payment import utils as payment_utils
+from odoo.addons.payment_test.controllers.main import PaymentTestController
+from odoo.addons.payment.tests.common import PaymentCommon
+
+
+class PaymentTestCommon(PaymentCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.acquirer = cls._prepare_acquirer(provider='test', update_values={
+            'capture_manually': False, # Only activate manual capture in dedicated tests
+            'fees_active': False, # Only activate fees in dedicated tests
+        })
+
+        cls.PaymentTestController = PaymentTestController()
+
+        cls.done_token = cls.env['payment.token'].create({
+            'acquirer_id': cls.acquirer.id,
+            'name': payment_utils.build_token_name(payment_details_short='4756'),
+            'partner_id': cls.partner.id,
+            'acquirer_ref': 'fake acquirer reference',
+            'verified': True,
+            'test_state': 'done',
+        })
+
+        cls.pending_token = cls.env['payment.token'].create({
+            'acquirer_id': cls.acquirer.id,
+            'name': payment_utils.build_token_name(payment_details_short='4756'),
+            'partner_id': cls.partner.id,
+            'acquirer_ref': 'fake acquirer reference',
+            'verified': True,
+            'test_state': 'pending',
+        })
+
+        cls.error_token = cls.env['payment.token'].create({
+            'acquirer_id': cls.acquirer.id,
+            'name': payment_utils.build_token_name(payment_details_short='4756'),
+            'partner_id': cls.partner.id,
+            'acquirer_ref': 'fake acquirer reference',
+            'verified': True,
+            'test_state': 'error',
+        })
+
+        cls.cancel_token = cls.env['payment.token'].create({
+            'acquirer_id': cls.acquirer.id,
+            'name': payment_utils.build_token_name(payment_details_short='4756'),
+            'partner_id': cls.partner.id,
+            'acquirer_ref': 'fake acquirer reference',
+            'verified': True,
+            'test_state': 'cancel',
+        })
```

Comment:
I would prefer a separate test file & class with this part of the setup & the tests related to tokenization :eyes: